processor/processor.go

```diff
@@ -309,7 +323,7 @@ func printLanguages() {
 	})
 
 	for _, name := range names {
-		fmt.Println(fmt.Sprintf("%s (%s)", name, strings.Join(database[name].Extensions, ",")))
+		fmt.Println(fmt.Sprintf("%s (%s)", name, strings.Join(append(database[name].Extensions, database[name].FileNames...), ",")))
```

Comment:
This use of `append` makes me cringe a little bit. I know `printLanguages` is outside the main loop and the arrays aren't too big, but this is still allocating new backing arrays and copying the elements for each invocation.

Reply:
Yeah I am with you, however keep in mind FileNames is empty most of the time as well.

I guess it could change to have a if condition where it needs to do this, but seems like additional work for no real benefit as you mention this almost never runs. 

Happy to modify it if you can think of something cleaner.

Reply:
Yeah, I know it's not in a performance critical code path and the arrays are tiny -- but it still makes me cringe.

Anyway, I can't think of a simple way of doing it without requiring extra code, so I'm not going to argue over it.

Reply:
Cool going to resolve this then.